fe/app/components/Quote/Quote.models.tsx

```diff
@@ -0,0 +1,11 @@
+export interface QuoteElementProps {
+  container?: { className: string };
+  text: {
+    content: React.JSX.Element;
+    author?: string;
+  };
```

Comment:
To simplify could you remove the nested props and update to something like this:
```
  className?: string;
  content: React.JSX.Element;
  author?: string;
```

Reply:
done